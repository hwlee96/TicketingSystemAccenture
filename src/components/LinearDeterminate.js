import React from "react";
import PropTypes from "prop-types";
import { withStyles } from "@material-ui/core/styles";
import LinearProgress from "@material-ui/core/LinearProgress";

const styles = {
  root: {
    flexGrow: 1
  }
};

class LinearDeterminate extends React.Component {
  render() {
    const { classes } = this.props;
    return (
      <div className={classes.root}>
        <LinearProgress variant="determinate" value={this.props.progress} />
        {/* <br />
        <LinearProgress color="secondary" variant="determinate" value={this.state.completed} /> */}
      </div>
    );
  }
}

LinearDeterminate.propTypes = {
  classes: PropTypes.object.isRequired
};

export default withStyles(styles)(LinearDeterminate);
