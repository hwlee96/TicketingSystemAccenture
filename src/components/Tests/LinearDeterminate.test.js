import React from "react";
import renderer from "react-test-renderer";
import { shallow, mount, render, configure } from "enzyme";
import Adapter from "enzyme-adapter-react-16";

import LinearDeterminate from "../LinearDeterminate";

configure({ adapter: new Adapter() });

it("renders correctly", () => {
  const tree = renderer.create(<LinearDeterminate />).toJSON();
  expect(tree).toMatchSnapshot();
});
