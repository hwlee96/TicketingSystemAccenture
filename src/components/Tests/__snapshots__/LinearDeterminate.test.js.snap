// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="LinearDeterminate-root-1"
>
  <div
    className="MuiLinearProgress-root-2 MuiLinearProgress-colorPrimary-3 MuiLinearProgress-determinate-5"
    role="progressbar"
  >
    <div
      className="MuiLinearProgress-bar-12 MuiLinearProgress-barColorPrimary-13 MuiLinearProgress-bar1Determinate-16"
      style={Object {}}
    />
  </div>
</div>
`;
